python/pyspark/sql/tests.py

```diff
@@ -1168,7 +1168,7 @@ def test_simple_udt_in_df(self):
         df = self.spark.createDataFrame(
             [(i % 3, PythonOnlyPoint(float(i), float(i))) for i in range(10)],
             schema=schema)
-        df.show()
+        df.collect()
```

Comment:
cc @viirya since this is added [here](https://github.com/apache/spark/commit/146001a9ffefc7aaedd3d888d68c7a9b80bca545#diff-7c2fe8530271c0635fb99f7b49e0c4a4R583).

Reply:
LGTM

Reply:
haha it's not a big deal tho.